tests/scripts/local_monitoring_tests/Local_monitoring_tests.md

```diff
@@ -182,3 +182,13 @@ Else, you can change the workload name and namespace name in the test to match w
 
 Note: The test will fail if it's run as is if there are no matching workloads that the test looks for. This test result can be ignored in case of a non-gpu workload
 
+### Authentication Test:
+
+Kruize 0.1.1 supports the authentication which provides the user an option to pass authentication details in the yaml for the service they are using.
```

Comment:
This should be 0.2 instead of 0.1.1. Also describe the scenarios added here

Reply:
Done